tests/CMakeLists.txt

```diff
@@ -241,7 +241,13 @@ add_test_case(test_byte_cursor_compare_lexical)
 add_test_case(test_byte_cursor_compare_lookup)
 
 add_test_case(byte_swap_test)
-add_test_case(alignment_test)
+
+if (HAVE_AVX2_INTRINSICS AND HAVE_SIMD_CPUID)
+    add_test_case(alignment32_test)
+else()
+    add_test_case(alignment16_test)
```

Comment:
Will 16 always work? Isn't the guarantee only 8?

Reply:
Henso suggested 16 which is what we've seen consistently on the failed internal builds.  Maybe dial back to 8 only if we have to.